app/src/pages/inside/common/accordion/accordionTab/accordionTab.scss

```diff
@@ -28,6 +28,7 @@
   font-size: 16px;
   text-transform: initial;
   cursor: pointer;
+  user-select: none;
```

Comment:
Why are we blocking user selection here?

Reply:
looks not so good if a user click on tab twice and it's leads to selecting tabs title